packages/parse5-test-utils/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why move this? E.g., `bench` is also a package, but not placed in `packages/`.

When I now look in `packages/`, in part due to the location and also its name, `parse6-test-utils` sounds like something that’s published to npm and consumable for others.

Maybe a different name, or move it elsewhere?

Reply:
Good point, I will move it back.

Reply:
Done